src/components/register/registerForm.js

```diff
@@ -69,15 +72,19 @@ export default function RegisterForm() {
       userType: e.target.value,
     });
   };
-  const handleSubmit = (e) => {
+  const handleSubmit = async (e) => {
     e.preventDefault();
-    dispatch(
+    await dispatch(
       postRegister({
         email: values.email,
         password: values.password,
         user_type: values.userType,
       })
     );
+    await postLogin({ email: values.email, password: values.password });
```

Comment:
Awesome job so that users won't have to login manually after registration. Great for the user experience.

Reply:
thanks, I hate when a site makes me do both!!